lib/cogctl/actions/roles/util.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Cogctl.Actions.Roles.Util do
+  alias Cogctl.Table
+  alias CogApi.Resources.Role
+  import Cogctl.ActionUtil , only: [display_output: 1, display_error: 1]
+
+  def render({:error, error}), do: display_error(error)
+  def render({info, _}), do: Table.format(info, false) |> display_output
+
+  def render(role_info, sort) do
+    Table.format(role_info, sort) |> display_output
+  end
+
+  def render(%Role{}=role, groups, permissions) do
+    role_info = format_role(role)
+    group_info = format_groups(groups)
+    permission_info = format_permissions(permissions)
+
+    Table.format(role_info ++ permission_info ++ group_info, false)
+    |> display_output
+  end
+
+  def render(role_info, sort, message) do
+    message <> "\n\n" <> Table.format(role_info, sort) |> display_output
+  end
+
+  defp format_role(nil), do: []
```

Comment:
We shouldn't need this function head. Role should always be available. If it isn't we have a bug.